release-scripts/getUpdatedVersions.js

```diff
@@ -0,0 +1,108 @@
+const { spawn } = require("child_process");
+const fs = require('fs/promises');
+const process = require('process')
+
+async function runProcess(command, args) {
+    const p = spawn(command, args)
+    let output = '';
+    p.stdout.on("data", data => {
+        output += data.toString()
+    });
+
+
+    return new Promise((resolve, reject) => {
+        p.on('exit', (code) => {
+            if (code !== 0) {
+                return reject(code);
+            }
+
+            resolve(output)
+        })
+    });
+}
+
+const VERSION_DIFF_REGEX = /^\+\s*"version"\s*:\s*"([^"]+)".*$/ //      '+  "version": "1.4.17",'
+const VERSION_DIFF_OLD_REGEX = /^\-\s*"version"\s*:\s*"([^"]+)".*$/ //   '-  "version": "1.4.16",'
```

Comment:
With support for things like `-beta.0`, correct?

Reply:
Yes, this will support it too.